src/main/scala/mesosphere/marathon/core/task/termination/TaskKillConfig.scala

```diff
@@ -0,0 +1,41 @@
+package mesosphere.marathon.core.task.termination
+
+import org.rogach.scallop.ScallopConf
+import scala.concurrent.duration._
+
+import scala.concurrent.duration.FiniteDuration
+
+trait TaskKillConfig extends ScallopConf {
+  //scalastyle:off magic.number
```

Comment:
if you put the magic number comment as a line comment after the code, it disables only for the line.